src/main/java/org/spongepowered/common/command/sponge/SpongeCommand.java

```diff
@@ -564,6 +572,31 @@ private TextColor tpsColor(final double tps) {
         return CommandResult.success();
     }
 
+    public static String formatMilisDate(Duration d) {
+        long days = d.toDays();
+        d = d.minusDays(days);
+        long hours = d.toHours();
+        d = d.minusHours(hours);
+        long minutes = d.toMinutes();
+        d = d.minusMinutes(minutes);
+        long seconds = d.getSeconds();
+        return
+                (days ==  0?"":days+" days, ") +
+                (hours == 0?"":hours+" hours, ") +
+                (minutes ==  0?"":minutes+" minutes, ") +
+                (seconds == 0?"":seconds+" seconds");
```

Comment:
this formatting needs some fixing

Reply:
Unsure how I could make this formatting better. :P

Reply:
fixed in d9b7d10